package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ng-iotile-cloud",
-  "version": "0.13.0",
+  "version": "0.12.10",
```

Comment:
Please note that I said before we should start modifying the MINOR number, and not the PATCH, so 0.13.0 was correct using this.